// myenergi-api/dist/src/models/Types.js
"use strict";
Object.defineProperty(exports, "__esModule", { value: true });
exports.ZappiStatus = exports.ZappiChargeMode = exports.ZappiBoostMode = exports.EddiMode = exports.EddiHeaterStatus = exports.EddiBoost = void 0;
var EddiBoost;
(function (EddiBoost) {
    EddiBoost["CancelHeater1"] = "1-1";
    EddiBoost["CancelHeater2"] = "1-1";
    EddiBoost["CancelRelay1"] = "1-11";
    EddiBoost["CancelRelay2"] = "1-12";
    EddiBoost["ManualHeater1"] = "10-1";
    EddiBoost["ManualHeater2"] = "10-1";
    EddiBoost["ManualRelay1"] = "10-11";
    EddiBoost["ManualRelay2"] = "10-12";
})(EddiBoost = exports.EddiBoost || (exports.EddiBoost = {}));
var EddiHeaterStatus;
(function (EddiHeaterStatus) {
    EddiHeaterStatus[EddiHeaterStatus["Starting"] = 0] = "Starting";
    EddiHeaterStatus[EddiHeaterStatus["Paused"] = 1] = "Paused";
    EddiHeaterStatus[EddiHeaterStatus["DSR"] = 2] = "DSR";
    EddiHeaterStatus[EddiHeaterStatus["Diverting"] = 3] = "Diverting";
    EddiHeaterStatus[EddiHeaterStatus["Boost"] = 4] = "Boost";
    EddiHeaterStatus[EddiHeaterStatus["Hot"] = 5] = "Hot";
    EddiHeaterStatus[EddiHeaterStatus["Stopped"] = 6] = "Stopped";
})(EddiHeaterStatus = exports.EddiHeaterStatus || (exports.EddiHeaterStatus = {}));
var EddiMode;
(function (EddiMode) {
    EddiMode[EddiMode["Off"] = 0] = "Off";
    EddiMode[EddiMode["On"] = 1] = "On";
})(EddiMode = exports.EddiMode || (exports.EddiMode = {}));
var ZappiBoostMode;
(function (ZappiBoostMode) {
    ZappiBoostMode[ZappiBoostMode["Manual"] = 10] = "Manual";
    ZappiBoostMode[ZappiBoostMode["Smart"] = 11] = "Smart";
    ZappiBoostMode[ZappiBoostMode["Stop"] = 2] = "Stop";
})(ZappiBoostMode = exports.ZappiBoostMode || (exports.ZappiBoostMode = {}));
var ZappiChargeMode;
(function (ZappiChargeMode) {
    ZappiChargeMode[ZappiChargeMode["Off"] = 4] = "Off";
    ZappiChargeMode[ZappiChargeMode["Fast"] = 1] = "Fast";
    ZappiChargeMode[ZappiChargeMode["Eco"] = 2] = "Eco";
    ZappiChargeMode[ZappiChargeMode["EcoPlus"] = 3] = "EcoPlus";
})(ZappiChargeMode = exports.ZappiChargeMode || (exports.ZappiChargeMode = {}));
var ZappiStatus;
(function (ZappiStatus) {
    ZappiStatus["EvDisconnected"] = "A";
    ZappiStatus["EvConnected"] = "B1";
    ZappiStatus["WaitingForEv"] = "B2";
    ZappiStatus["EvReadyToCharge"] = "C1";
    ZappiStatus["Charging"] = "C2";
    ZappiStatus["Fault"] = "F";
})(ZappiStatus = exports.ZappiStatus || (exports.ZappiStatus = {}));
